### DISCUSSION TITLE
Falsche Minus Temperatur auf dem Dashboard

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP_Buderus_KM271#51] [General]
![Aussentemp](https://github.com/dewenni/ESP_Buderus_KM271/assets/47242804/fcfa4922-7194-4200-a78e-4e978ed1e8c7)

Da hat sich ein kleiner Fehler eingschlichten.

Grüße
Manfred

**Comment by dewenni** [comment]
Hallo Manfred,

danke für den Hinweis. Habe den Fehler gefunden und werde es mit dem nächsten Release beheben.

Grüße Sven

**Comment by Kenaschon** [comment]
Hallo Sven,

danke für die schnelle Reaktion.
